src/components/Executions.js

```diff
@@ -244,20 +258,32 @@ class Executions extends Component {
           className={classes.link}
           onClick={() => this.handleResultView(resultId)}
         >
-          {resultId}
+          {resultName}
         </Button>
       );
 
       let statusIcon = null;
       switch (status) {
         case EXECUTION_STATUSES.SUCCESS:
-          statusIcon = <SuccessIcon style={{ color: 'green' }} />;
+          statusIcon = (
+            <Tooltip title={t('Successful execution')}>
+              <SuccessIcon style={{ color: 'green' }} />
+            </Tooltip>
+          );
           break;
         case EXECUTION_STATUSES.ERROR:
-          statusIcon = <ErrorIcon style={{ color: 'red' }} />;
+          statusIcon = (
+            <Tooltip title={t('An error occured')}>
+              <ErrorIcon style={{ color: 'red' }} />
+            </Tooltip>
+          );
           break;
         case EXECUTION_STATUSES.RUNNING:
-          statusIcon = <RunningIcon style={{ color: 'orange' }} />;
+          statusIcon = (
+            <Tooltip title={t('The execution is running...')}>
+              <CircularProgress />
```

Comment:
add `size={26}` (or thereabouts, 26 looked good to me). but the default is definitely a bit big